src/components/molecules/card/CardHeader.tsx

```diff
@@ -1,21 +1,41 @@
-import { Box } from '@material-ui/core';
+import { Box, createStyles, makeStyles, Theme } from '@material-ui/core';
 import React, { FC } from 'react';
 import { HeaderVariant } from '../../../services/widgets.style.service';
 import RoadNumberImage from './RoadNumberImage';
 import LamasImage from '../../../assets/cbs.png';
 import AnywayImage from '../../../assets/anyway.png';
 import { Typography, Logo } from '../../atoms';
 import { useTranslation } from 'react-i18next';
+import { silverSmokeColor } from '../../../style/_globals';
 
 const MAX_WORDS_PER_TITLE_LINE = 5;
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    labelWrapper: {
+      backgroundColor: silverSmokeColor,
+      maxWidth: '10rem',
```

Comment:
Should be constant - use `width` instead of `maxWidth`
Also, card sizes are in px - use `cardWidth` const for relative size (for example: `cardWidth / 3`